fair_mango/metrics/base.py

```diff
@@ -35,12 +35,12 @@ def is_binary(y: pd.Series | pd.DataFrame) -> bool:
 
 
 def encode_target(data: Dataset, ind: int, col: str):
```

Comment:
suggestion: add None return type. I saw it multiple time, I stop adding comments related to this issue